docs/messaging/messaging-metrics.md

```diff
@@ -88,6 +119,18 @@ the broker doesn't have such notion, the destination name SHOULD uniquely identi
 | `_OTHER` | A fallback error value to be used when the instrumentation doesn't define a custom value. | ![Stable](https://img.shields.io/badge/-stable-lightgreen) |
 
 
+`messaging.operation.type` has the following list of well-known values. If one of them applies, then the respective value MUST be used; otherwise, a custom value MAY be used.
+
+| Value  | Description | Stability |
+|---|---|---|
+| `create` | A message is created. "Create" spans always refer to a single message and are used to provide a unique creation context for messages in batch publishing scenarios. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `deliver` | Deprecated. Use `process` instead. | ![Deprecated](https://img.shields.io/badge/-deprecated-red)<br>Replaced by `process`. |
```

Comment:
Do we need this? It might make sense to have it in the attribute registry, but I think it's confusing to have it (indefinitely?) in the conventions.

Reply:
there is no way to not have it 🤐

Reply:
I see you submitted https://github.com/open-telemetry/semantic-conventions/pull/1110, which will take care of this. Thanks!